tests/ludwig/schema/test_model_config.py

```diff
@@ -634,3 +636,50 @@ def test_number_feature_zscore_preprocessing_default():
     config_obj = ModelConfig.from_dict(config)
 
     assert config_obj.input_features.number_input_feature1.preprocessing.normalization == "zscore"
+
+
+@pytest.mark.parametrize(
+    "augmentation,expected",
+    [
+        (None, []),
```

Comment:
@tgaddair This is fine, though are you sure we wanna support `None` as an option?

Reply:
This is a bit misleading. If you look further down, you'll see that `None` here actually means "don't include in the config".

Reply:
ah, my bad glanced over that `del`. thanks!